src/components/objects/Player.js

```diff
@@ -11,14 +11,11 @@ class Player extends Phaser.Group {
     constructor(data, game) {
         super(game, null, 'Player', true, true);
 
-        // this._direction = game..directions.RIGHT;
-        this._moveTimer = 0;
-
         for (const bodyPart of data._bodyParts) {
             this.expandBody({
                 x: bodyPart._x,
                 y: bodyPart._y,
-            });
+            }, data._color);
```

Comment:
We could store the color in this player object before this call is made, but I don't see the use of that. 

What you are looking at is the actual player running this code in its constructor, and the way I see it is that, since the player itself does not need the color(it's supposed to be used by the individual instances of BodyPart), we just pass the network payload further down the scope. The payload in that regard is the "data" variable, meaning containers of information that have yet not been used to create something useful.